src/config/recorder_config.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <simdjson.h>
```

Comment:
头文件里没有用到的话，只放在cc里就好

Reply:
Used in the `ConfigDataParser` declaration

Reply:
@zptan per [Google C++ style guide]([Enable Tidy Checks and CI](https://github.com/cyfitech/cris-core/pull/110/files#top)

> Avoid using forward declarations where possible. Instead, [include the headers you need](https://google.github.io/styleguide/cppguide.html#Include_What_You_Use).
> 
> Pros
> 
> - Forward declarations can save compile time, as #includes force the compiler to open more files and process more input.

As you mentioned. Also

> - Forward declarations can save on unnecessary recompilation. #includes can force your code to be recompiled more often, due to unrelated changes in the header.

But

> Cons
>
> - Forward declarations can hide a dependency, allowing user code to skip necessary recompilation when headers change.
> - A forward declaration as opposed to an #include statement makes it difficult for automatic tooling to discover the module defining the symbol.
> - A forward declaration may be broken by subsequent changes to the library. Forward declarations of functions and templates can prevent the header owners from making otherwise-compatible changes to their APIs, such as widening a parameter type, adding a template parameter with a default value, or migrating to a new namespace.
> - Forward declaring symbols from namespace std:: yields undefined behavior.
It can be difficult to determine whether a forward declaration or a full #include is needed. Replacing an #include with a forward declaration can silently change the meaning of code:

```
// b.h:
struct B {};
struct D : B {};

// good_user.cc:
#include "b.h"
void f(B*);
void f(void*);
void test(D* x) { f(x); }  // Calls f(B*)
```
> If the #include was replaced with forward decls for B and D, test() would call f(void*).
> - Forward declaring multiple symbols from a header can be more verbose than simply #includeing the header.
> - Structuring code to enable forward declarations (e.g., using pointer members instead of object members) can make the code slower and more complex.